tasks/queue-granules/src/index.ts

```diff
@@ -181,6 +181,7 @@ async function queueGranules(event: HandlerEvent): Promise<QueueGranulesOutput>
                   updatedAt: updatedAt ?? createdAt,
                   createdAt: createdAt,
                   archived: archived || false,
+                  active: 'A',
```

Comment:
Why would queue granules set this to 'A'? 

I think our downstream duplicate task should probably make that decision.    Default write should be to active, but I don't think a task should be opinionated about that.